deploy/crds/org_v1_che_crd-v1beta1.yaml

```diff
@@ -601,17 +601,28 @@ spec:
                   type: object
                 devfileRegistryUrl:
                   description: Public URL of the devfile registry, that serves sample,
-                    ready-to-use devfiles. Set this ONLY when a use of an external
-                    devfile registry is needed. See the `externalDevfileRegistry`
-                    field. By default, this will be automatically calculated by the
-                    Operator.
+                    ready-to-use devfiles. This will be automatically calculated by
+                    the Operator. See the `externalDevfileRegistry` and `externalDevfileRegistries`
+                    fields.
                   type: string
+                externalDevfileRegistries:
+                  description: External devfile registries, that serves sample, ready-to-use
+                    devfiles. See the `externalDevfileRegistry` field.
+                  items:
+                    description: Settings forconfiguration of the external devfile
+                      registries.
+                    properties:
+                      url:
+                        description: Public URL of the devfile registry.
+                        type: string
+                    type: object
+                  type: array
                 externalDevfileRegistry:
                   description: Instructs the Operator on whether to deploy a dedicated
                     devfile registry server. By default, a dedicated devfile registry
                     server is started. When `externalDevfileRegistry` is `true`, no
```

Comment:
Is it true? I thought the purpose of PR was that add an ability to use external registry when internal (managed by operator) is still present.